src/include/replication/replication_messages.h

```diff
@@ -3,8 +3,11 @@
 #include <limits>
 #include <memory>
 #include <string>
+#include <utility>
+#include <vector>
 
 #include "common/enum_defs.h"
+#include "common/json.h"
```

Comment:
ah... just saw the templated Put/Get. ok, this will look like ass. but I think it is worth the compilation times. can you move the template declarations to the .cpp file and explicitly instantiate them for the types that we use? sorry

Reply:
Fixed